src/components/non-bitgo.js

```diff
@@ -117,6 +117,10 @@ class NonBitGoRecoveryForm extends Component {
         return obj;
       }, {});
 
+      if(this.state.coin === 'btc'&& this.state.apiKey) {
```

Comment:
Sorry about bothering about whitespaces, but at least in 4 places the formatting is off.

Reply:
And colons are missing on some lines.